src/app/withdraw/components/withdraw-client.tsx

```diff
@@ -87,6 +88,16 @@ const Withdraw = () => {
     },
   });
 
+  const {
+    data: emilyLimits,
+    refetch,
+    isLoading: emilyLimitsLoading,
+  } = useEmilyLimits();
+
+  const maxWithdrawal = (emilyLimits?.perWithdrawalCap || 0) / 1e8;
```

Comment:
Is `emilyLimits` guaranteed to have a value, even if we just called `useEmilyLimits`?

Reply:
no it's not guaranteed until the action/request is fetched from the server